README.md

```diff
@@ -20,12 +20,49 @@ Any exceptions to this are noted in the individual `README.md` files.
 
 ## Installation
 
-TBD
+The repository layout structure is as defined by [the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md). The `api`
+directory includes header-only abstract classes and no-op implementations that comprise the OpenTelemetry API following
+[the
+specification](https://github.com/open-telemetry/opentelemetry-specification).
+The `sdk` folder is the reference implementation of the API.
+
+The library would be distributed in source-only format. No shared/static library or distribution package (rpm, deb etc) would be made available as part of Release. See [RELEASING.md](./RELEASING.md).
+
+This project supports `Bazel` and `CMake` builds.
+
+### Using CMake
+On most platforms with all dependencies installed, the following command will compile the source files, and install the header files:
+
+```sh
+cmake -Bcmake-out
+cmake --build cmake-out
+sudo cmake --build cmake-out --target install
+
+#To run the tests
+cd cmake-out && ctest
+```
 
-## Quick Start
+In case the library is distributed through package manager, below is the recommended deployment directory structure for header and librarie(s) on Linux and MacOS platforms.
 
+```
+/usr/local/lib:  For shared or static opentelemetry-cpp library
+/usr/local/include/opentelemetry: For api headers
+/usr/local/include/opentelemetry/sdk: For sdk headers
+/usr/local/src: For installing source package ( say libopentelemetry-dev )
```

Comment:
I think this related to a previous discussion. However, I don't think we should put it here with this wording. We might need to document where `make install` puts our libraries, but we shouldn't recommend deployment structures for package managers.

Any proper DEB or RPM package will not install in `/usr/local`, but in `/usr/include` and `/usr/lib`. We should leave it to package maintainers to figure those things out (as I think we can't give valid general advice here).

Reply:
Thanks Johannes. I do somewhat agree with your comment.  @maxgolov  your thought : )

Reply:
Ideally it should say `$INSTALL_PREFIX` instead of `/usr` - as this is either `/usr` or `/usr/local` . Maybe we should remove this and handle in a separate installation document, where we can describe the layout more precisely, depending on target OS and package manager: for rpm, deb, brew, vcpkg and others most common package managers. For CMake - we should also list what variables are populated by `find_package`, such as - include and library dirs of the opentelemetry package.

Reply:
> For CMake - we should also list what variables are populated by find_package, such as - include and library dirs of the opentelemetry package(s).

Do we intend to provide a CMake integration so the library can be discovered via `find_package`? We don't have this yet.

I'm fine with providing this directory list under the label of _default_ (CMake) locations, I'm a bit confused by labeling this as _recommended deployment directory structure_, as recommendations might vary considerable depending on the use case.

Reply:
@pyohannes - I can work on this for CMake.

Reply:
Have removed all the install information, and just kept a reference to ./INSTALL.md. Hope that is fine.